Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Action/ActionManager.cpp

```diff
@@ -242,6 +242,19 @@ namespace AzToolsFramework
         return AZ::Success();
     }
 
+    ActionManagerBooleanResult ActionManager::IsActionEnabled(const AZStd::string& actionIdentifier)
```

Comment:
question: Could this be marked `const`? (even if this is an EBus interface it's now generally good practice to add const correctness where possible as we may use this through an AZ::Interface)

Reply:
Makes sense, fixed!